### ISSUE TITLE
How can I set some space between Input and Icon properly?

### ISSUE THREAD

**Issue by sereymongkol** [react-native-elements/react-native-elements#2099]
I am new to both react native and react native elements and I have read react native elements documents as well as test some code provides in the docs but the result is not the same as shown in the document page. I have tried on this [section](https://react-native-training.github.io/react-native-elements/docs/input.html) and then this is the code I tried: 
```
 <View>
     <Input
         placeholder='PASSWORD'
         style={{padding: 50}}
          // leftIcon={ <Icon name="lock" size={24} style={{padding:}}/> } // this one is working but imagine if I have some reason not to use <Icon/>
          leftIcon={{ type: 'font-awesome', name: 'lock' }} />
  </View>
```
Here is the result i got: 

![Screenshot_1570032226](https://user-images.githubusercontent.com/53964371/66061697-26b1ee80-e56a-11e9-873e-164d94706588.png)

The output look too bad compare to the one i saw on the documents. So, how can I set some space between icon and input? 

I almost forget. i believe what shown on document(code, picture) should be relevant so we can easily copy and paste and furthermore, there should be more code example. I used to use bootstrap, this provide many example including form ... And if this project could do that too, it would be much easier for new comer to learn and copycat. Thanks.

**Comment by iRoachie** [comment]
This is a bug that's to be fixed in 2.0.0, we didn't fix it earlier cause it would've been a breaking change.

However, we have a prop called [`leftIconContainerStyle`](https://react-native-training.github.io/react-native-elements/docs/input.html#lefticoncontainerstyle) that you can use to remove the margin. See example https://snack.expo.io/@roach_iam/intelligent-pizza

**Comment by sereymongkol** [comment]
@iRoachie , wow it works like charm! Anyway, is there any beta version I can use with bug fixed? I will use beta if there is because putting this everywhere is not the best solution as later on i will need to delete it unless the gap is too narrow.

**Comment by iRoachie** [comment]
Not as yet, we do have it planned for the next release. See https://github.com/react-native-elements/react-native-elements/issues/1962

**Comment by sereymongkol** [comment]
Hello @iRoachie, can I ask you something? I saw on the main Github page, the screenshots look awesome. Buttons have many color, round or square shape and many style but when I tried the provided code on the docs page, thing is so simple. They doesn't contain anything or can say almost identical to default react native components. So, is that I do it wrong or it is like that? And if it is so, then how can i find more example like in the screenshot? thanks

**Comment by iRoachie** [comment]
The screenshots are from our example app https://github.com/react-native-elements/react-native-elements-app

All the code is provided

**Comment by goxr3plus** [comment]
I am using :

```
 <Button
     title="Add Goal"
     textStyle={{ marginLeft: 20 }}
     icon={<Icon name="plus" size={15} color="white" />}
     onPress={addGoalHandler}
  />
```

And there is still no space between button text and icon.

**Comment by flyingcircle** [comment]
This should be fixed as of now since v2.0.0. Changing the padding should be as simple as overriding the `leftIconContainerStyle`

**Comment by cglacet** [comment]
> This should be fixed as of now since v2.0.0. Changing the padding should be as simple as overriding the `leftIconContainerStyle`.

Could you elaborate on that, I tried the following but it seems to have no effect (version **2.3.2**): 


```
<Button
    title={'Submit'}
    leftIconContainerStyle={{ margin: 20 }}
    icon={<FontAwesome5 name="check" size={15} color="white" />}
/>
```

On the other hand, the following does work: 

```
<Button
    title={'Submit'}
    icon={<FontAwesome5 name="check" size={15} color="white" style={{ margin: 20 }} />}
/>
```

**Comment by flyingcircle** [comment]
@cglacet this ticket was about `Input` not `Button`. I suggest reviewing the docs to know what props are available. https://react-native-elements.github.io/react-native-elements/docs/button#iconcontainerstyle
